integration/spark/src/main/scala/org/apache/spark/sql/catalyst/CarbonParserUtil.scala

```diff
@@ -677,6 +677,8 @@ object CarbonParserUtil {
         val errorMsg = "range_column not support multiple columns"
         throw new MalformedCarbonCommandException(errorMsg)
       }
+      CommonUtil.validateForSpatialTypeColumn(tableProperties, rangeColumn,
```

Comment:
Instead of checking property by property, once all the properties are filled, better to validate at one place  ?

Also I see that for sortcolumns, column_metacache,no_inverted_index it is not handled

Reply:
modified the method to validate all table properties in one place as suggested.